cachito/web/content_manifest.py

```diff
@@ -31,13 +37,14 @@
 class ContentManifest:
     """A content manifest associated with a Cachito request."""
 
-    def __init__(self, request):
+    def __init__(self, request, packages):
         """
         Initialize ContentManifest.
 
         :param Request request: the request to generate a ContentManifest for
         """
         self.request = request
+        self.packages = [Package.from_json(package) for package in packages]
```

Comment:
Ideally, `ContentManifest` should not have the responsibility to convert a `package` dict to a `Package` object. It just consumes the `Package` objects to get purl and other content manifest.

It would be good to convert in the `content_manifest` property instead.

Reply:
I did just that, take a look now :wink: